bcgov/HOUS-permit-portal
# optimize draft search and hooks

## Description
search optimization for permit applications
## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-888

## Steps to QA

navigate to and fro on the submitter permit application search and reviewer inbox. check the network tab to see search requests. Navigate back and forth from single to list permit applications
